src/coffee/io/writer.coffee

```diff
@@ -0,0 +1,105 @@
+_ = require 'underscore'
+Csv = require 'csv'
+path = require 'path'
+Promise = require 'bluebird'
+iconv = require 'iconv-lite'
+fs = Promise.promisifyAll require('fs')
+Excel = require 'exceljs'
+
+DEBUG = false
+debugLog = if DEBUG then console.log else _.noop
```

Comment:
I agree - on whole project there is no special logger used, just plain `console`. If we decide to use logger we should add it also to other parts of the project so this was just to quickly add at least something.